uni/lib/view/bug_report/widgets/form.dart

```diff
@@ -276,22 +276,26 @@ class BugReportFormState extends State<BugReportForm> {
       Logger().i('Successfully submitted bug report.');
       if (context.mounted) toastMsg = S.of(context).success;
       status = true;
-    } catch (e) {
+    } catch (e, stackTrace) {
+      await Sentry.captureException(e, stackTrace: stackTrace);
```

Comment:
Isn't it best to use `unawaited` when capturing exceptions to sentry? I don't really know if it takes a lot of time when a connection is slow, but if it does, this would mean the app would not block when sending the exception to sentry and it would be done in the background instead.

Reply:
Well, this does not run in the UI thread (it's a callback) so I don't think it makes much of a difference here